plugins/terminal-proxy/lib/ssh.js

```diff
@@ -1288,12 +1288,16 @@ function readSSHv2PDUData(sessionData,rawData,currentPosition){
     var receivedMac = Buffer.alloc(sessionData.readingMACLength);
     rawData.copy(receivedMac,0,currentPosition);
     currentPosition+=receivedMac.length;
-    if (receivedMac.toString('hex')!=computedMac.toString('hex')){
-      sshLogger.warn("mac verification failed. ");
+    if (computedMac == undefined) {
+      sshLogger.warn('mac verification failed')
+    } else {
+      if (receivedMac.toString('hex')!=computedMac.toString('hex')){
+        sshLogger.warn("mac verification failed. ");
+      }
+      if(traceCrypto) console.log("received mac: "+receivedMac.toString('hex'));
+      if(traceCrypto) console.log("computed mac: "+computedMac.toString('hex'));
```

Comment:
Can you please put both `console.log` statements under a single `if (traceCrypto)`.